### DISCUSSION TITLE
Issue installingnew version of  waybar to solve hyprland workspace display issue

### DISCUSSION THREAD

**Discussion by coacharnold1** [JaKooLit/Hyprland-Dots#284] [Q&A]
Hello

Using Fedora 40. 

I lost my workspaces selection section on waybar. Ive found a solution on here to install the most upto date version from git-hub Waybar 0.10.3    ... but I can't get it to install.  

The make files use Meson Build and Ninja,  both I have never encountered.  Ive installed both and cannot get them to run without an error.   Has anyone else had this problem and can you help?

Thanks
Tim

**Comment by JaKooLit** [comment]
This should be actually to waybar developer instead of here..

But then somebody already ask this 

https://github.com/Alexays/Waybar/issues/3238

anyway, sees like maintainer went to vacation lmao


make sure you have depencies installed...

depencies are all in waybar github

building

meson build
sudo ninja -C build install

reboot
